cmd/metricsexporter/metricsExporter.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"flag"
+	"net"
+	"net/http"
+	"os"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/prometheus/common/promlog"
+	promflag "github.com/prometheus/common/promlog/flag"
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+var (
+	exporterNamespace string
+	exporterAddress   string
+	kubeconfigPath    string
+	kubeAPIServerURL  string
+	verbose           bool
+)
+
+type lvmCollector struct{}
+
+func (lc *lvmCollector) Describe(c chan<- *prometheus.Desc) {}
+func (lc *lvmCollector) Collect(c chan<- prometheus.Metric) {}
+
+func main() {
+	lvmExpFlags := flag.NewFlagSet("lvm-metrics-exporter", flag.ExitOnError)
+	lvmExpFlags.StringVar(&exporterNamespace, "namespace", "lvm-exporter",
```

Comment:
Where is used the namespace?

Reply:
Currently this metric exporter is a dummy one which don't export/expose any particular metrics. Once this is actively used then we require this flag, otherwise this is a place holder